### PR TITLE
SELFDEV-370: [update] Replace links

### PR DESCRIPTION
**Pull request by slsriehl**
# [SELFDEV-370]

## What changed?
Provide a bulleted list in the present tense
* Replace links

## Anything else?
Add related PRs, salient notes, ticket numbers, etc.

ping @becomevocal @bc-stephen-hilliard 

[SELFDEV-370]: https://bigcommercecloud.atlassian.net/browse/SELFDEV-370?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/api-docs/storefront/scripts-overview.mdx [hunk 1]
```diff
@@ -69,7 +69,7 @@ For more on checking which theme engine a storefront is using, see this article'
 
 ## Managing scripts using single-click app callbacks
 
-If you're using an [app-level API account](/api-docs/getting-started/api-accounts#app-level-api-accounts), we recommend inserting your scripts immediately following completion of the [auth callback](/api-docs/apps/guide/auth). The app can use the [uninstall callback](/api-docs/apps/guide/callbacks) to remove its scripts when a merchant uninstalls it, and the [remove_user callback](/api-docs/apps/guide/callbacks) to remove any scripts related to a particular user when they lose access privileges.
+If you're using an [app-level API account](/api-docs/getting-started/api-accounts#app-level-api-accounts), we recommend inserting your scripts immediately following completion of the [auth callback](/api-docs/apps/guide/auth). The app can use the [uninstall callback](/docs/integrations/apps/guide/callbacks) to remove its scripts when a merchant uninstalls it, and the [remove_user callback](/api-docs/apps/guide/callbacks) to remove any scripts related to a particular user when they lose access privileges.
```
Comment (bc-stephen-hilliard) [inline docs/api-docs/storefront/scripts-overview.mdx:72]: these 2 `/api-docs/apps/guide/auth` & `/api-docs/apps/guide/callbacks` I am getting different links to ie `/docs/integrations/apps/guide/auth` & `/docs/integrations/apps/guide/callbacks`

**Review by bc-stephen-hilliard — CHANGES_REQUESTED** [review]

File: docs/stencil-docs/deploying-a-theme/troubleshooting-theme-uploads.mdx [hunk 2]
```diff
@@ -48,7 +48,7 @@ Warnings will not block a theme's upload, but these onscreen and/or log messages
 | (These messages do not have numeric codes) |
 | Issue in processing this theme's thumbnail screenshot (`composed_image`). |
 | Issue in processing this theme's full-size screenshot (`desktop_screenshot`). |
-| Issue in processing this theme's mobile [screenshot](/stencil-docs/stencil-theme-editor/config-json-metadata) (`mobile_screenshot`). |
+| Issue in processing this theme's mobile [screenshot](/theme-objects/config-json#configjson) (`mobile_screenshot`). |
```
[the remark this answers is not in the document]
Comment (bc-traciporter) [reply docs/stencil-docs/deploying-a-theme/troubleshooting-theme-uploads.mdx:51]: I am getting this path: /docs/storefront/stencil/themes/context/object-reference/config#configjson

File: docs/stencil-docs/developing-further/customizing-emails.mdx [hunk 3]
```diff
@@ -4,7 +4,7 @@
 This article provides information on how to create your own unique email templates using our legacy framework. 
 
 <Callout type="info">
- For information on customizing email templates on our latest theme framework engine, see [Customizing Emails](https://support.bigcommerce.com/s/article/Customizing-Emails) and [Handlebars Email Template Objects](/api-docs/store-management/email-templates/handlebars).
+ For information on customizing email templates on our latest theme framework engine, see [Customizing Emails](https://support.bigcommerce.com/s/article/Customizing-Emails) and [Handlebars Email Template Objects](/docs/rest-content/email-template-objects).
```
[the remark this answers is not in the document]
Comment (bc-traciporter) [reply docs/stencil-docs/developing-further/customizing-emails.mdx:7]: same ^

**Review by bc-traciporter — COMMENTED** [review]
see comments

File: docs/legacy/storefronts/nextjs-commerce-guide.mdx [hunk 4]
```diff
@@ -165,7 +165,7 @@ SWR uses FETCH for data fetching: [vercel/fetch: Opinionated fetch (with retryin
 
 ### SSL/TSL Certificate
 
-To use BigCommerce's [redirected checkout](https://developer.bigcommerce.com/api-docs/storefronts/guide/checkout#redirecting-to-the-bigcommerce-checkout), make sure your Next.js Commerce storefront has an SSL/TSL certificate installed and you are using URL beginning with `https://`.
+To use BigCommerce's [redirected checkout](https://developer.bigcommerce.com/docs/storefront/headless/cart-checkout/checkout#redirecting-to-the-bigcommerce-checkout), make sure your Next.js Commerce storefront has an SSL/TSL certificate installed and you are using URL beginning with `https://`.
```
Comment (slsriehl) [inline docs/legacy/storefronts/nextjs-commerce-guide.mdx:168]:
```suggestion
To use BigCommerce's [redirected checkout](/docs/storefront/headless/cart-checkout/checkout#redirecting-to-the-bigcommerce-checkout), make sure your Next.js Commerce storefront has an SSL/TSL certificate installed and you are using URL beginning with `https://`.
```

File: docs/api-docs/getting-started/rest-api-authentication.mdx [hunk 5]
```diff
@@ -239,7 +239,7 @@ All OAuth scopes except `default` provide `read-only` permissions scopes so that
 
 | UI Name | Permission | Parameter | Description | Resources |
 |:--------|:-----------|:----------|:------------|:----------|
-| Account | read-only | `account_read` | View account details | [GraphQL Account API](/docs/graphql-users) |
+| Account | read-only | `account_read` | View account details | [GraphQL Account API](/docs/graphql-account/users-overview) |
```
[the remark this answers is not in the document]
Comment (slsriehl) [reply docs/api-docs/getting-started/rest-api-authentication.mdx:242]: resolved

File: docs/api-docs/management-apis/email-templates/overview.mdx [hunk 6]
```diff
@@ -9,7 +9,7 @@ This article provides an overview of BigCommerce's transactional email templates
 
 To edit transactional email templates in a store's control panel, [log in](https://support.bigcommerce.com/s/article/Logging-Into-Your-Store) and navigate to **Marketing** › [**Transactional Emails**](http://login.bigcommerce.com/deep-links/manage/transactional-emails).
 
-Use [Handlebars expressions](/api-docs/store-management/email-templates/handlebars) to localize email templates and access dynamic data (like a customer's first name in an order email).
+Use [Handlebars expressions](/docs/rest-content/email-template-objects) to localize email templates and access dynamic data (like a customer's first name in an order email).
```
[the remark this answers is not in the document]
Comment (slsriehl) [reply docs/api-docs/management-apis/email-templates/overview.mdx]: fixed

File: docs/api-docs/storefront/scripts-overview.mdx (hunk 1 above)
Comment (slsriehl) [reply docs/api-docs/storefront/scripts-overview.mdx]: fixed i believe

File: docs/stencil-docs/developing-further/customizing-emails.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (slsriehl) [reply docs/stencil-docs/developing-further/customizing-emails.mdx]: as it should, and this link has been updated.

[the remark this answers is not in the document]
Comment (slsriehl) [reply docs/stencil-docs/developing-further/customizing-emails.mdx]: also a redirect instituted

File: docs/stencil-docs/deploying-a-theme/troubleshooting-theme-uploads.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (slsriehl) [reply docs/stencil-docs/deploying-a-theme/troubleshooting-theme-uploads.mdx]: correct and i believe this is fixed

**Pull request merged**